api/src/wfl/module/covid.clj

```diff
@@ -595,12 +593,33 @@
         (update :fromSource edn/read-string))
     (throw (ex-info "Invalid executor_items" {:workload workload}))))
 
-(defn verify-terra-executor!
+(defn verify-terra-executor
   "Verify the method-configuration exists."
-  [{:keys [name methodConfiguration skipValidation] :as executor}]
+  [{:keys [skipValidation
+           workspace
+           methodConfiguration
+           methodConfigurationVersion
+           fromSource] :as executor}]
   (when-not skipValidation
-    (when-not (:methodConfiguration executor)
-      (throw (ex-info "Unknown Method Configuration" {:executor executor})))))
+    (when-not (do-or-nil (firecloud/workspace workspace))
+      (throw
+       (UserException. "Cannot access workspace" (util/make-map workspace))))
+    (when-not (= "importSnapshot" fromSource)
+      (throw
+       (UserException. "Unsupported coercion" (util/make-map fromSource))))
+    (if-let [{:keys [methodConfigVersion]}
```

Comment:
Does this block make the checks in `update-method-configuration!` superfluous?

Reply:
nope - they're still valid I think